tests/testutils/test_package_to_lint.py

```diff
@@ -0,0 +1,48 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+from pathlib import Path
+
+from pylint.testutils.primer import PRIMER_DIRECTORY_PATH, PackageToLint
+
+
+def test_package_to_lint() -> None:
```

Comment:
Do we want to test the cloning functionality?

Reply:
I added a comment and a pragma, what do oyu think ?
```python
    def lazy_clone(self) -> None:  # pragma: no cover
        """Concatenates the target directory and clones the file

        Not expected to be tested as the primer won't work if it doesn't. 
        We'll notice."""
```

Reply:
Fair enough!

Reply:
It's tested in the primer, only the coverage is not calculated on everything. And if it breaks for local use with lazy clone we'll probably notice because we'll have a fatal when launching the primer. (Might want to add that to the comment 😄)